mail_tracking/tests/test_mail_tracking.py

```diff
@@ -0,0 +1,706 @@
+# Copyright 2016 Antonio Espinosa - <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import base64
+import time
+from unittest.mock import patch
+
+from werkzeug.exceptions import BadRequest
+
+from odoo import http
+from odoo.fields import Command
+from odoo.tests.common import TransactionCase
+from odoo.tools.misc import mute_logger
```

Comment:
```suggestion
from odoo.tools import mute_logger
```

(as it was in previous reviewed version)